onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -447,78 +200,62 @@ Status Attention<T>::Compute(OpKernelContext* context) const {
   const Tensor* past = context->Input<Tensor>(4);
   const Tensor* extra_add_qk = context->Input<Tensor>(5);
 
+  const Tensor* key = context->Input<Tensor>(6);
```

Comment:
Is "key" used? I just found it's only one of the inputs to the check_input()

Reply:
Not used in CPU EP yet. Just for input checking right now.